frontend/appflowy_flutter/lib/plugins/document/presentation/more/more_button.dart

```diff
@@ -24,7 +25,7 @@ class DocumentMoreButton extends StatelessWidget {
         child: FlowyHover(
           child: Padding(
             padding: const EdgeInsets.all(6),
-            child: FlowySvg(
+            child: ScalableFlowySvg(
```

Comment:
I suggest adding a new parameter, such as scaleFactor, to `FlowySvg` instead of creating a new `ScalableFlowySvg`. The current implementation will increase complexity.

Reply:
`ScalableFlowySvg` reply on `AppearanceSettingsCubit` to get the current `fontIconsSizeFactor`. If we add `scale` in `FlowySvg`, the  `FlowySvg` reply on `AppearanceSettingsCubit` to get the current `fontIconsSizeFactor`? or get `fontIconsSizeFactor` from `AppearanceSettingsCubit` every time before building `FlowySvg`?